File: CS/OneMinuteDragDrop/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraScheduler;
using DevExpress.XtraScheduler.Drawing;

namespace OneMinuteDragDrop {
    public partial class Form1 : Form {
        AppointmentDragHelper dragHandler;
        AppointmentResizeHelper resizeHandler;

        public Form1() {
            InitializeComponent();
            schedulerControl1.DayView.AppointmentDisplayOptions.SnapToCellsMode = AppointmentSnapToCellsMode.Never;
            schedulerControl1.WorkWeekView.AppointmentDisplayOptions.SnapToCellsMode = AppointmentSnapToCellsMode.Never;

            dragHandler = new AppointmentDragHelper(schedulerControl1);
            resizeHandler = new AppointmentResizeHelper(schedulerControl1);
        }

        private void checkEdit1_CheckedChanged(object sender, EventArgs e) {
            if (((CheckEdit)sender).Checked)
                dragHandler.AttachToControl();
            else
                dragHandler.DetachFromControl();
        }

        private void checkEdit2_CheckedChanged(object sender, EventArgs e) {
            if (((CheckEdit)sender).Checked)
                resizeHandler.AttachToControl();
            else
                resizeHandler.DetachFromControl();
        }
    }
}

File: VB/OneMinuteDragDrop/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraScheduler
Imports DevExpress.XtraScheduler.Drawing

Namespace OneMinuteDragDrop
    Partial Public Class Form1
        Inherits Form

        Private dragHandler As AppointmentDragHelper
        Private resizeHandler As AppointmentResizeHelper

        Public Sub New()
            InitializeComponent()
            schedulerControl1.DayView.AppointmentDisplayOptions.SnapToCellsMode = AppointmentSnapToCellsMode.Never
            schedulerControl1.WorkWeekView.AppointmentDisplayOptions.SnapToCellsMode = AppointmentSnapToCellsMode.Never

            dragHandler = New AppointmentDragHelper(schedulerControl1)
            resizeHandler = New AppointmentResizeHelper(schedulerControl1)
        End Sub

        Private Sub checkEdit1_CheckedChanged(ByVal sender As Object, ByVal e As EventArgs) Handles checkEdit1.CheckedChanged
            If DirectCast(sender, CheckEdit).Checked Then
                dragHandler.AttachToControl()
            Else
                dragHandler.DetachFromControl()
            End If
        End Sub

        Private Sub checkEdit2_CheckedChanged(ByVal sender As Object, ByVal e As EventArgs) Handles checkEdit2.CheckedChanged
            If DirectCast(sender, CheckEdit).Checked Then
                resizeHandler.AttachToControl()
            Else
                resizeHandler.DetachFromControl()
            End If
        End Sub
    End Class
End Namespace